packages/create-cloudflare/src/templates.ts

```diff
@@ -132,7 +132,7 @@ export type TemplateConfig = {
 		ctx: C3Context,
 	) => Promise<Record<string, string | object>>;
 
-	/** An array of compatibility flags to be specified when deploying to pages or workers.*/
+	/** An array of compatibility flags to be specified when deploying to pages (unused for workers) */
```

Comment:
Good spot. I see that a couple of other Workers based c3.ts files still include this property. We should remove those as well.
E.g. 
- packages/create-cloudflare/templates-experimental/solid/c3.ts
- packages/create-cloudflare/templates/hello-world-with-assets/c3.ts

Reply:
https://github.com/cloudflare/workers-sdk/pull/9210